pkg/image-manager/image_builder.go

```diff
@@ -122,16 +111,11 @@ func (b *BaseImageBuilder) dockerDelete(baseImage *BaseImage) error {
 	if err != nil && baseImage.Status == StatusREADY {
 		return errors.Wrapf(err, "Error deleting image %s/%s", baseImage.OrganizationID, baseImage.Name)
 	}
-	var deleted BaseImage
-	err = b.es.Delete(b.orgID, baseImage.Name, &deleted)
-	if err != nil {
-		return errors.Wrapf(err, "Error deleting base image entity %s/%s", baseImage.OrganizationID, baseImage.Name)
-	}
 	log.Printf("Successfully deleted base image %s/%s", baseImage.OrganizationID, baseImage.Name)
 	return nil
 }
 
-func (b *BaseImageBuilder) dockerStatus() ([]*BaseImage, error) {
+func (b *BaseImageBuilder) dockerStatus() ([]entitystore.Entity, error) {
```

Comment:
I think it is better to be named as baseImageStatus, baseImageDelete and baseImagePull for BaseImageBuilder if you used imageStatus, imageDelete and imagePull for ImageBuilder